src/Tizen.NUI.Components/Controls/Navigation/AppBar.cs

```diff
@@ -255,10 +257,11 @@ public View TitleContent
         /// <summary>
         /// Action views of AppBar.
         /// Action views are added to ActionContent of AppBar.
-        /// The Action and ActionButton styles of AppBarStyle are applied to actions only by setting Actions.
-        /// If you do not want to apply Action and ActionButton styles to action views, then please use ActionContent.Add() instead.
+        /// If action views and action buttons are set to Actions, then proper style look for action views and action buttons are automatically applied to action viws and action buttons.
```

Comment:
viws -> views

Reply:
Thank you for your comment.
The description has been fixed.